Packs/Mattermost/Integrations/MattermostV2/MattermostV2.yml

```diff
@@ -521,12 +538,114 @@ script:
       - 'channel'
     description: Mirrors the investigation between Mattermost and the Cortex XSOAR War Room.
     name: mattermost-mirror-investigation
+  - arguments:
+    - name: page
+      description: The page number to retrieve. Default value is 0.
+    - name: page_size
+      description: The size of the page to retrieve. Default value is 50.
+    - name: limit
+      description: How many results to retrieve. Will override the page and page_size
+        arguments if given.
+    - name: group
+      description: Search for a specific group by this pattern.
+    name: mattermost-list-groups
+    outputs:
+    - contextPath: Mattermost.Groups.id
+      description: The ID of the group.
+      type: String
+    - contextPath: Mattermost.Groups.create_at
+      description: When was the group created.
+      type: Unknown
+    - contextPath: Mattermost.Groups.update_at
+      description: When was the group updated.
+      type: Unknown
+    - contextPath: Mattermost.Groups.delete_at
+      description: When was the group deleted.
+      type: Unknown
+    - contextPath: Mattermost.Groups.display_name
+      description: The display name of the group.
+      type: String
+    - contextPath: Mattermost.Groups.name
+      description: The name of the group.
+      type: String
+    - contextPath: Mattermost.Groups.description
+      description: The description of the group.
+      type: String
+    - contextPath: Mattermost.Groups.source
+      description: The source of the group.
+      type: String
+    - contextPath: Mattermost.Groups.remote_id
+      description: The remote id of the group.
+      type: String
+    - contextPath: Mattermost.Groups.has_syncables
+      description: If the group has any syncables.
+      type: boolean
+    description: 'Lists groups. Groups are available on Enterprise and Professional plans. Minimum server version: 5.11'
+  - name: mattermost-list-group-members
```

Comment:
Let's add pagination support since the [endpoint](https://api.mattermost.com/#tag/groups/operation/GetGroupUsers) supports pagination